web_dashboard_tile/tile.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+       <record model="ir.ui.view" id="dashboard_tile_tile_tree_view">
+            <field name="name">tile.tile.tree</field>
+            <field name="model">tile.tile</field>
+            <field name="type">tree</field>
```

Comment:
This line is useless. (and 2 other below)
Cf https://runbot.odoo-community.org/runbot/static/build/2924792-54-6f0bec/logs/job_20_test_all.txt.